portfolio.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="portfolio.css">
+</head>
+<body>
+  <header>
+    <a href="" class="nameLink">John Doe</a>
+    <nav>
+      <img src="images/Icon-Menu.png" alt="Menu Icon" class='menuIcon'>
+    </nav>
+  </header>
+  <section>
+    <h1>Hey There. I'm Alex</h1>
+    <h2>I’am a Software Developer</h2>
+    <p class="firstParagraph">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+    <ul class="Icon">
+      <li><img src="images/Icon-GitHub.png" alt="Github Icon" class="Icons Icon1"></li>
+      <li><img src="images/Icon-Linkedin.png" alt="Linkedin Icon" class="Icons Icon2"></li>
+      <li><img src="images/Icon-Angelist.png" alt="Angelist Icon" class="Icons Icon3"></li>
+      <li><img src="images/Icon-Twitter.png" alt="Twitter Icon" class="Icons Icon4"></li>
+      <li><img src="images/Icon-Medium.png" alt="Medium Icon" class="Icons Icon5"></li>
```

Comment:
Please wrap every icon image with a link tag so that it can be a clickable link.